templates/zerver/digest_base.html

```diff
@@ -1,6 +1,10 @@
 {% extends "zerver/base.html" %}
 {% set entrypoint = "digest" %}
 
+{% block title %}
+<title>Zulip digest</title>
```

Comment:
Did not mark this to be translated. Also, I wasn't sure if following my general rule of "Zulip: ..." would be preferred for this page. That would make the title "Zulip: Digest".

Reply:
Updated to be tagged for translation with the text as "Digest | Zulip".